lib/elixir/lib/macro.ex

```diff
@@ -763,11 +763,11 @@ defmodule Macro do
     to_string(ast, fun)
   end
 
-  defp bitmods_to_string({:-, _, [left, right]} = ast, fun, _, _) do
+  defp bitmods_to_string({atom, _, [left, right]} = ast, fun, _, _) when atom in [:*, :-] do
     result =
-      bitmods_to_string(left, fun, :-, :left) <>
-      "-" <>
-      bitmods_to_string(right, fun, :-, :right)
+      bitmods_to_string(left, fun, atom, :left) <>
+      "#{atom}" <>
```

Comment:
Should i use `Atom.to_string(atom)` here for performance reasons? or it doesn't really matter